### PR TITLE
Make Wintertodt messages consistent with minionKill

### LINKED ISSUE #4887: Incorrect Grammar/Inconsistent formatting on some food items
## 🐛 Bug Report

This is related to starting a trip and having the food item used be Chili potatoes. 

Generally when you use other potatoes (e.g. Potato with Cheese), the formatting is **Removed ##x Potato with cheese** (in bold). When you use Chili Potatoes, though, it isn't bolded and it uses an apostrophe to pluralize the word potatoes (potato's).

See screenshots:
image
image

Anyway, it's a small thing, but an apostrophe doesn't pluralize words and the plural of potato is "potatoes."

### PR DESCRIPTION
**Pull request by osrsjobbo**
### Description:

This closes #4887. I took inspiration from minionKill.ts to try create a consistent experience. 

image

### Changes:

Use line breaks & bold to make more consistent with other areas of the bot. 

### Other checks:

-   [X] I have tested all my changes thoroughly.

### CODE REVIEW

File: src/mahoji/lib/abstracted_commands/wintertodtCommand.ts [hunk 1]
```diff
@@ -23,7 +23,11 @@ export async function wintertodtCommand(user: MUser, channelID: string) {
 
 	// Up to a 10% boost for 99 WC
 	const wcBoost = (wcLevel + 1) / 10;
-	if (wcBoost > 1) messages.push(`${wcBoost.toFixed(2)}% boost for Woodcutting level`);
+
+	if (wcBoost > 1) {
+		messages.push(`**Boosts: ** ${wcBoost.toFixed(2)}%  for Woodcutting level\n`);
```
Comment (gc) [inline src/mahoji/lib/abstracted_commands/wintertodtCommand.ts:28]: this string is written weirdly, you're making a space bold - and theres a double space

**Review by gc — CHANGES_REQUESTED** [review]

**Comment by osrsjobbo** [comment]
@gc Done & tested

**Comment by gc** [comment]
Failing tests

**Comment by osrsjobbo** [comment]
Fixed

**Pull request merged**